### ISSUE TITLE
ArrayTags: Could not connect to the processor

### ISSUE THREAD

**Issue by shakthimaan** [jogibear9988/ControlLogixNET#1]
Hi,

I am trying out the ArrayTags example with an Allen-Bradley hardware 1769-L24ER-QB1B/A LOGIX5324ER, revision 20.1.

The program fails in the processor.Connect() method and displays "Could not connect to the processor."

I tried packet captures with Wireshark and the "Register Session" and "List Identity" requests receive responses, but, the "Forward Open" request fails with 'Connection failure' with status, 'Extended: 0x0103'.

Any idea what could be the problem?

**Comment by jogibear9988** [comment]
No, I dont even have a Alan Bradley Plc! I only forked the project, because I create a Visualisation app, where I support AB Cpus, and on Codeplex the development stopped!
If you find out, i will include the patch...
